pkg/webreg/webreg.go

```diff
@@ -838,6 +839,26 @@ func syntaxBash(source string) (string, error) {
 	return syntax(source, lexers.Get("bash"))
 }
 
+func fromImage(name string, reference *api.ImageStreamTagReference) string {
```

Comment:
That request is unrelated to your change, but could you move `getBaseTemplate` into an `init` that populates a `var` so that:
* We don't parse the template over and over
* We notice when its broken at unit test time (by making the `init` panic)
* We won't crash the binary when the template is broken and someone visits a page that uses it because in that case we currently just log an error and return a nilpointer: https://github.com/openshift/ci-tools/blob/64c968a7395db4f3dce9706d90ba23cf902fdb41/pkg/webreg/webreg.go#L540-L544

Reply:
SGTM